prometheus/prometheus.rules

```diff
@@ -5,5 +5,15 @@ ALERT InstanceDown
   LABELS { severity = "1" }
   ANNOTATIONS {
     summary = "Instance {{ $labels.instance }} down",
-    description = "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 30 seconds.",
+    description = "{{ $labels.instance }} has been down for more than 30 seconds.",
+  }
+
+# Alert for any instance that it's free disk space bellow 5%.
```

Comment:
The number 5 is used 3 times in the code.
- Can it be extracted to a var?
- 25% make more sense

Reply:
I couldn't find var support in the rule files.
I'll change to 25%